lib/sqlalchemy/orm/mapped_collection.py

```diff
@@ -38,21 +50,26 @@ class _PlainColumnGetter:
 
     __slots__ = ("cols", "composite")
 
-    def __init__(self, cols):
+    def __init__(self, cols: List[ColumnElement[Optional[_KT]]]) -> None:
```

Comment:
can this be Sequence not List

Reply:
I have fixed this and related issues in latest commit, could you please take a look?